lib/infoblox/resource/grid.rb

```diff
@@ -0,0 +1,63 @@
+# Taken from https://github.com/IMPIMBA/infoblox
+# Credit where credit is due.
+#
+module Infoblox
+  class Grid < Resource
+    remote_attr_reader   :audit_to_syslog_enable,
+                         :external_syslog_server_enable,
+                         :password_setting,
+                         :security_banner_setting,
+                         :security_setting,
+                         :snmp_setting,
+                         :syslog_facility,
+                         :syslog_servers,
+                         :syslog_size
+
+    wapi_object "grid"
+
+    def delete
+      raise "Not supported"
+    end
+
+    def create
+      raise "Not supported"
+    end
+
+    def modify
+      raise "Not supported"
+    end
+
+    def self.get(connection)
+      JSON.parse(connection.get(resource_uri).body).map do |jobj|
+        klass = resource_map[jobj["_ref"].split("/").first]
+        if klass.nil?
+          puts jobj['_ref']
```

Comment:
This actually looks like a good case for raising an exception to me.  I would remove the `puts` and change the `warn` to a `raise` with a `RuntimeError`.